src/models/request.ts

```diff
@@ -1,39 +1,33 @@
-import { RequestStatus } from './request-status.js'
-import TypeORM from 'typeorm'
-const { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn, Unique } =
-  TypeORM
+export const TABLE_NAME = 'request'
+
+// How long we should retain requests that were completed or failed
+export const ANCHOR_DATA_RETENTION_WINDOW = 1000 * 60 * 60 * 24 * 30 // 30 days
```

Comment:
This seems like a weird place for these constants?  They aren't really related to the Request structure, but are really about the behavior of the request repository and anchor service

Reply:
I debated about this. The problem I was trying to solve was that these constants are littered throughout the code and are kind of hard to find unless you know the code well. Ex. READY_TIMEOUT was in anchor-service.ts while PROCESSING_TIMEOUT was in request-repo. 

So while I agree this may not be the right place for these constants I would like to make sure we keep them in a consistent place thats easy to find. Maybe I should put them all in settings.ts?

Reply:
I'll actually do this in a follow up PR